app/src/main/AndroidManifest.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="org.resolvetosavelives.red"
-  xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  package="org.resolvetosavelives.red">
 
   <uses-permission android:name="android.permission.CAMERA" />
   <uses-permission android:name="android.permission.VIBRATE" />
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
We did not have internet permissions so far? :O how was patient sync working then?